openapi/endpoints/funding-sources/models/funding-source-interaction.yaml

```diff
@@ -0,0 +1,77 @@
+type: object
+properties:
+  id:
+    type: string
+    description: id of the Funding Source Interaction.
+    example: f0220f084a182e3f4b4d605cda1d3340
+  fundingSourceId:
+    type: string
+    description: Funding Source to which the interaction belongs to.
+    example: 2be03cd65da12fe29b8910af7b4b85e7
+  accountId:
+    type: string
+    description: The accountId to which the interaction belongs to.
+    example: 979254377d0e05002242d038926a5691
+  type:
+    type: string
+    description: The interaction type.
+    enum: 
+      - Deposit
+      - Withdrawal
+      - Refund
+      - Payment
+    example: Payment
+  status:
+    type: string
+    description: The current status of the interaction. It will vary depending on the interaction type.
```

Comment:
we should probably explain what the different common statuses mean

Reply:
what are the common statuses?